test/apps/iodemo/io_demo.cc

```diff
@@ -1253,8 +1263,14 @@ class DemoClient : public P2pDemoCommon {
                 return;
             }
 
+            size_t data_size = 0;
+
+            for (int i = 0; i < _iov->size(); ++i) {
+                data_size += (*_iov)[i].size();
+            }
```

Comment:
common function to calc iov total size?

Reply:
can be method of `BufferIov` and canclulated on initialization

Reply:
BTW, it's already there, just need to store `data_size`

Reply:
good catch, done